packages/core/src/hooks/form/useForm.ts

```diff
@@ -378,6 +386,10 @@ export const useForm = <
     const { mutate: mutateUpdate, isLoading: isLoadingUpdate } =
         mutationResultUpdate;
 
+    const mutationAutoSave = useUpdate<TResponse, TResponseError, TVariables>(
```

Comment:
```suggestion
    const autoSaveMutation = useUpdate<TResponse, TResponseError, TVariables>(
```

I think it's slightly better.